packages/terra-slide-panel-manager/src/terra-dev-site/doc/example/SlidePanelManagerSquishExample.jsx

```diff
@@ -0,0 +1,307 @@
+            },
+          });
+        }}
+      >
+        {`Disclose (${size})`}
+      </button>
+    );
+  }
+
+  renderFormButton() {
+    const { disclosureType } = this.props;
+    const disclosureManager = this.context;
+
+    const name = `Disclose (${this.state.disclosureHeight}) x (${this.state.disclosureWidth})`;
+
+    return (
+      <button
+        type="button"
+        onClick={() => {
+          disclosureManager.disclose({
+            preferredType: disclosureType,
+            dimensions: { height: this.state.disclosureHeight, width: this.state.disclosureWidth },
+            content: {
+              key: 'Content-Disclosure-Dimensions',
+              component: <DisclosureComponent name="Disclosure Component" disclosureType={disclosureType} />,
+            },
+          });
+        }}
+      >
+        {name}
+      </button>
+    );
+  }
+
+  renderForm() {
+    return (
+      <form>
+        <label htmlFor={this.getId(`disclosureHeight${this.props.panelBehavior}`)}>Pop Content Height</label>
+        <select id={this.getId(`disclosureHeight${this.props.panelBehavior}`)} name="disclosureHeight" value={this.state.disclosureHeight} onChange={this.handleSelectChange}>
+          {generateDimensionOptions(HEIGHT_KEYS)}
+        </select>
+        <br />
+        <br />
+        <label htmlFor={this.getId(`disclosureWidth${this.props.panelBehavior}`)}>Pop Content Width</label>
+        <select id={this.getId(`disclosureWidth${this.props.panelBehavior}`)} name="disclosureWidth" value={this.state.disclosureWidth} onChange={this.handleSelectChange}>
+          {generateDimensionOptions(WIDTH_KEYS)}
+        </select>
+        <br />
+        <br />
+      </form>
+    );
+  }
+
+  render() {
+    return (
+      <ContentContainer
+        header={(
+          <ActionHeader
+            title="SlidePanelManager Child Component"
+          />
+        )}
+      >
+        <div className={cx('content-wrapper')}>
+          {this.renderButton('default')}
+          {this.renderButton('tiny')}
+          {this.renderButton('small')}
+          {this.renderButton('medium')}
+          {this.renderButton('large')}
+          {this.renderButton('huge')}
+          {this.renderButton('fullscreen')}
+          <br />
+          <p>The child components can disclose content in the panel at various sizes.</p>
+          <p>The sizes available are limited to those of the SlidePanel. The specified size/dimensions will be mapped to the nearest available size.</p>
+        </div>
+        <div className={cx('content-wrapper')}>
+          {this.renderForm()}
+          {this.renderFormButton()}
+        </div>
+      </ContentContainer>
+    );
+  }
+}
+
+ContentComponent.contextType = DisclosureManagerContext;
+ContentComponent.propTypes = {
+  disclosureType: PropTypes.string,
+  panelBehavior: PropTypes.oneOf(['overlay', 'squish']),
+};
+
+const SlidePanelManagerExample = ({ behavior, showAccessory }) => (
```

Comment:
Lets remove the props from this example & write specifically for the Squish example